src/PDPVerifier.sol

```diff
@@ -481,6 +477,11 @@ contract PDPVerifier is Initializable, UUPSUpgradeable, OwnableUpgradeable {
     function nextProvingPeriod(uint256 setId, uint256 challengeEpoch, bytes calldata extraData) public {
         require(extraData.length <= EXTRA_DATA_MAX_SIZE, "Extra data too large");
         require(msg.sender == proofSetOwner[setId], "only the owner can move to next proving period");
```

Comment:
@ZenGround0 We should revert this function if prover calls it on an empty proofset.  Prover MUST make atleast one call to `addRoot` before calling this to get set up their first deadline to start proving.

Reply:
It does not make sense to start "proving" an "empty proofset".

Reply:
I like this